deploy-java/action.yml

```diff
@@ -138,6 +138,14 @@ runs:
         jiraUsername: ${{ inputs.jiraUsername }}
         jiraPassword: ${{ inputs.jiraPassword }}
 
+    - name: On success notify okay to test
+      uses: actions/github-script@v6
+      if: ${{ (inputs.environment !='prod') && (inputs.environment !='qa04') }}
```

Comment:
Could you add comment somewhere explaining 
`inputs.environment !='qa04'`
Seems out of place.